src/ProjectOrigin.Chronicler.Server/Repositories/ChroniclerRepository.cs

```diff
@@ -163,4 +163,22 @@ public async Task WithdrawClaimRecord(FederatedCertificateId fid)
         if (rowsChanged != 1)
             throw new InvalidOperationException($"ClaimRecord with registry {fid.RegistryName} and certificateId {fid.StreamId} not found");
     }
+
+    public async Task UnclaimClaimRecord(FederatedCertificateId fid)
+    {
+        var rowsChanged = await Connection.ExecuteAsync(
+            @"UPDATE claim_records
+                  SET state = @state
+                  WHERE registry_name = @registryName
+                    AND certificate_id = @certificateId",
+            new
+            {
+                registryName = fid.RegistryName,
+                certificateId = fid.StreamId,
+                state = ClaimRecordState.Unclaimed
+            });
+
+        if (rowsChanged != 1)
+            throw new InvalidOperationException($"ClaimRecord with registry {fid.RegistryName} and certificateId {fid.StreamId} not found");
+    }
```

Comment:
Hmm, just got to thinking, do we really want these sort of failures?
How would this handle transactions for certificates in a registry in reads, but not an area it tracks, this it would deadlock the processing, right?